pennylane/tape/plxpr_conversion.py

```diff
@@ -0,0 +1,236 @@
+
+    return []
+
+
+@CollectOpsandMeas.register_primitive(cond_prim)
+def _(self, *all_args, jaxpr_branches, consts_slices, args_slice):
+    n_branches = len(jaxpr_branches)
+    conditions = all_args[:n_branches]
+    args = all_args[args_slice]
+
+    # Find predicates that use mid-circuit measurements. We don't check the last
+    # condition as that is always `True`.
+    mcm_conditions = tuple(pred for pred in conditions[:-1] if isinstance(pred, MeasurementValue))
+    if mcm_conditions:
+        if len(mcm_conditions) != len(conditions) - 1:
+            raise ValueError(
+                "Cannot use qml.cond with a combination of mid-circuit measurements "
+                "and other classical conditions as predicates."
+            )
+        conditions = _get_mcm_predicates(mcm_conditions)
+
+    for pred, jaxpr, const_slice in zip(conditions, jaxpr_branches, consts_slices):
+        consts = all_args[const_slice]
+        if jaxpr is None:
+            continue
+        if isinstance(pred, qml.measurements.MeasurementValue):
+            if jaxpr.outvars:
+                outvals = [v.aval for v in jaxpr.outvars]
+                raise ValueError(
+                    (
+                        "Conditional branches of mid circuit measurements are not allowed to"
+                        f" return anything with plxpr_to_tape and CollectOpsandMeas. Branch returns {outvals}"
+                    )
+                )
+            child = CollectOpsandMeas()
+            child.eval(jaxpr, consts, *args)
+            assert child.state
+            self.state["ops"].extend(qml.ops.Conditional(pred, op) for op in child.state["ops"])
+        elif pred:
+            return copy(self).eval(jaxpr, consts, *args)
+    return ()
+
+
+@CollectOpsandMeas.register_primitive(measure_prim)
+def _(self, wires, reset, postselect):
+    m0 = qml.measure(wires, reset=reset, postselect=postselect)
+    self.state["ops"].extend(m0.measurements)
+    return m0
+
+
+def plxpr_to_tape(jaxpr: "jax.core.Jaxpr", consts, *args, shots=None) -> QuantumScript:
+    """Convert a jaxpr into a tape.
+
+    Args:
+        jaxpr (jax.core.Jaxpr): a pennylane variant jaxpr
+        consts (list): the consts for the jaxpr
+        *args : the arguments to execute the jaxpr with
+
+    Keyword Args:
+        shots (None, int, Sequence[int], Shots): the shots for the tape.
+
+    Returns:
+        QuantumScript: a single quantum script containing the quantum operations and measurements
+
+    .. code-block:: python
+
+        @qml.for_loop(3)
+        def loop(i):
+            qml.X(i)
+
+        def f(x):
+            loop()
+            qml.adjoint(qml.S)(0)
+            m0 = qml.measure(0)
+            qml.RX(2*x, 0)
+            return qml.probs(wires=0), qml.expval(qml.Z(1))
+
+        jaxpr = jax.make_jaxpr(f)(0.5)
+        tape = qml.capture.convert_to_tape(jaxpr.jaxpr, jaxpr.consts, 1.2)
+        print(qml.drawer.tape_text(tape, decimals=2))
```

Comment:
```suggestion
        >>> from pennylane.tape.plxpr_conversion import plxpr_to_tape
        >>> qml.capture.enable()
        >>> jaxpr = jax.make_jaxpr(f)(0.5)
        >>> tape = plxpr_to_tape(jaxpr.jaxpr, jaxpr.consts, 1.2)
        >>> print(qml.drawer.tape_text(tape, decimals=2))
```

Reply:
```suggestion
       qml.capture.enable()

        jaxpr = jax.make_jaxpr(f)(0.5)
        tape = qml.capture.convert_to_tape(jaxpr.jaxpr, jaxpr.consts, 1.2)
        print(qml.drawer.tape_text(tape, decimals=2))
```

Adding the `qml.capture.enable()`,  but not sure why we want to explicitly import it instead of just using the `qml.tape.plxpr_to_tape` path.